git/week2/yasi/Smart-ease-secendproject.js

```diff
@@ -0,0 +1,156 @@
+//Flight booking fullname function
+let fullname1 = getFullname("Sarah", "Tomes");
+let fullname2 = getFullname("Alice", "Baker", false);
+function getFullname(firstname, surname, useFormalName = true) {
+  if (!firstname || !surname) {
+    return "Invalid name";
+  }
+  if (useFormalName === true) {
+    return "Lord" + " " + firstname + " " + surname;
+  } else {
+    return firstname + " " + surname;
+  }
+}
+// Testing the function
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+function getEventWeekday(daysFromToday) {
+  const days = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  const today = new Date();
+  const eventDay = (today.getDay() + daysFromToday) % 7;
+  return days[eventDay];
+}
+// Testing the function
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(0));
+
+//Weather wear
+function whatToWear(temperature) {
+  if (temperature >= 25) {
+    return "Wear shorts and a t-shirt";
+  } else if (temperature >= 15) {
+    return "Wear jeans and a light jacket";
+  } else if (temperature >= 5) {
+    return "Wear a warm jacket, sweater, and pants";
+  } else {
+    return "Wear a heavy coat, gloves, and a hat";
+  }
+}
+
+// Testing the function
+const clothesToWear = whatToWear(18);
+console.log(clothesToWear);
+
+//Student manager
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+  if (studentName === "") {
+    console.log("Invalid name");
+    return;
+  }
+
+  if (class07Students.includes(studentName)) {
+    console.log("Student " + studentName + " is already in the class");
+    return;
+  }
+
+  if (studentName === "Queen") {
+    class07Students.push(studentName);
+    return;
+  }
+
+  if (class07Students.length > 6) {
```

Comment:
Change to
```suggestion
  if (class07Students.length >= 6) {
```
otherwise you would get 8 students instead of 7